src/i18n2.ts

```diff
@@ -0,0 +1,39 @@
+import { i18n } from 'i18next';
```

Comment:
Var ikkje tanken at du skulle importere i18ninstance fra ndla-ui og legge til lokale tekster i den? Eller misforstår eg då?

Reply:
Du har rett! Denne importen brukes kun for å få inn typen på `i18n`-parameteret. `i18nInstance` hentes inn i `client.jsx`.